kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/IfcfgConfigWriter.java

```diff
@@ -40,22 +38,21 @@
 import org.eclipse.kura.net.NetInterfaceConfig;
 import org.eclipse.kura.net.NetInterfaceStatus;
 import org.eclipse.kura.net.NetInterfaceType;
+import org.eclipse.kura.net.admin.visitor.linux.util.IfcfgConfig;
 import org.eclipse.kura.net.admin.visitor.linux.util.KuranetConfig;
 import org.eclipse.kura.net.wifi.WifiInterfaceAddressConfig;
 import org.eclipse.kura.net.wifi.WifiMode;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class IfcfgConfigWriter implements NetworkConfigurationVisitor {
+public class IfcfgConfigWriter extends IfcfgConfig implements NetworkConfigurationVisitor {
 
     private static final Logger logger = LoggerFactory.getLogger(IfcfgConfigWriter.class);
 
     private static final String REDHAT_NET_CONFIGURATION_DIRECTORY = "/etc/sysconfig/network-scripts/";
```

Comment:
Please remove it since it is defined in IfcfgConfig

Reply:
done